scripts/commands/updateApiDocs.ts

```diff
@@ -178,23 +176,30 @@ zxMain(async () => {
       `Invalid --version. Expected the format 0.44.0, but received ${args.version}`,
     );
   }
-  const versionWithoutPatch = versionMatch[0];
 
-  const pkg = PACKAGES.find((pkg) => pkg.name === args.package);
-  if (pkg === undefined) {
+  const pkgInfo = PACKAGES.find((pkg) => pkg.name === args.package);
+  if (pkgInfo === undefined) {
     throw new Error(`Unrecognized package: ${args.package}`);
   }
 
-  if (args.historical) {
+  const pkg: Pkg = {
+    version: args.version,
+    versionWithoutPatch: versionMatch[0],
+    historical: args.historical,
+    releaseNoteEntries: [],
+    ...pkgInfo,
+  };
+
+  if (pkg.historical) {
     if (pkg.name !== "qiskit") {
       throw new Error("`--historical` can only be used with `-p qiskit`");
     }
-    pkg.baseUrl = `https://qiskit.org/documentation/stable/${versionWithoutPatch}`;
+    pkg.baseUrl = `https://qiskit.org/documentation/stable/${pkg.versionWithoutPatch}`;
     pkg.initialUrls = [`${pkg.baseUrl}/apidoc/index.html`];
   }
 
   const destination = `${getRoot()}/.out/python/sources/${pkg.name}/${
-    args.version
+    pkg.version
   }`;
   if (await pathExists(destination)) {
     console.log(`Skip downloading sources for ${pkg.name}:${args.version}`);
```

Comment:
Nit: This line continues using `args` instead of `pkg`.
 
```suggestion
    console.log(`Skip downloading sources for ${pkg.name}:${pkg.version}`);
```